.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Build
+        run: yarn run build
```

Comment:
Does it make sense (and is it possible) to build the developer version as well, to validate that it also works?

Reply:
there is no developer version, the developer version is only served by a dev server in development. We could build a dev version, but it is not currently intended by the npm build config

Reply:
At the very least things like `import.meta.env.PROD` are different, and I believe optimization is also omitted

Reply:
But I think we can leave this be for now